main.cpp

```diff
@@ -459,24 +458,31 @@ struct Envelope
     float releaseTime = 675.1f;
     float curve = 0.9f;
 
-    void padPreset(float eCurve);
-    void staccatoPreset(float eCurve);
-    bool loopOver(float eAttackTime, float eDecayTime, float eReleaseTime);
+    void padPreset();
+    void staccatoPreset();
+    bool loopOver();
 };
 
-void Envelope::padPreset(float eCurve)
+void Envelope::padPreset()
 {
-    std::cout << "Setting envelope to slow setting, with curve of: " << eCurve << std::endl;
+    attackTime = 1500.f;
+    sustainLevel = 1.0;
+    releaseTime = 4500.f;
+    std::cout << "Setting envelope to slow setting, with curve of: " << curve << std::endl;
 }
-void Envelope::staccatoPreset(float eCurve)
+void Envelope::staccatoPreset()
 {
-    std::cout << "Setting envelope to fast setting, with curve of: " << eCurve << std::endl;
+    attackTime = 2.f;
+    decayTime = 300.f;
+    sustainLevel = 0.0;
+    releaseTime = 300.f;
+    std::cout << "Setting envelope to fast setting, with curve of: " << curve << std::endl;
 }
```

Comment:
These are bad function names.  There is no action happening. 
A better choice is
`loadPadPreset()` or `loadStaccatoPreset()`
- [ ] revise